### PR TITLE
Updated modules to latest version

### PR DESCRIPTION
**Pull request by JayXTQ**
All hashes should be correct, unsure about 7-Zip though as it took an arm and a leg to find the correct files needed

### CODE REVIEW

File: src/AtlasModules/README.md [hunk 1]
```diff
@@ -42,8 +42,8 @@ To verify the legitmacy of these binaries (executables), there's information pro
 - `serviwin.exe` was extracted from the `.zip`
 
 ### Vcredist
-- SHA256 Hash: `b67f9072d75000b184e27f62894c2e7ae8093541fc5a697393980b9c24b30ccb`
-- Source: https://github.com/abbodi1406/vcredist/releases/download/v0.60.0/VisualCppRedist_AIO_x86_x64_60.zip
+- SHA256 Hash: `A61F16097FCB9CAE2D4E203CED8FB817C59B29724DC0E9F1DC3981DE0588464D`
```
Comment (he3als) [inline src/AtlasModules/README.md:45]: Verified hash

File: src/AtlasModules/README.md [hunk 2]
```diff
@@ -2,17 +2,17 @@
...
-- Last Verified: 2022-03-05 by he3als
+- Last Verified: 2022-12-01 by JayXTQ
 - The executable was extracted from the 64-bit version.
 
 ### 7z
-- SHA256 Hash (`7z.exe`): `ED24ED04B5D4A20B3F50FC088A455195C756D7B5315D1965E8C569472B43D939`
-- SHA256 Hash (`7z.dll`): `405E9A77857753D21E060DCFFAB12F820FA7593AE152B45588703BF312B5FF44`
-- Source: https://sourceforge.net/projects/sevenzip/files/7-Zip/21.07/7z2107-x64.exe/download
-- Last Verified: 2022-03-05 by he3als
+- SHA256 Hash (`7z.exe`): `254CF6411D38903B2440819F7E0A847F0CFEE7F8096CFAD9E90FEA62F42B0C23`
```
Comment (he3als) [inline src/AtlasModules/README.md:12]: Verified hash

File: src/AtlasModules/README.md [hunk 3]
```diff
@@ -2,17 +2,17 @@
...
+- SHA256 Hash (`7z.dll`): `73578F14D50F747EFA82527A503F1AD542F9DB170E2901EDDB54D6BCE93FC00E`
```
Comment (he3als) [inline src/AtlasModules/README.md:13]: Verified hash

File: src/AtlasModules/README.md [hunk 4]
```diff
@@ -2,17 +2,17 @@
 To verify the legitmacy of these binaries (executables), there's information provided here for each binary for hashes, sources and when each was last verified/checked with the SHA256 hash. Verified using `Get-FileHash` in PowerShell. 
 
 ### DevManView
-- SHA256 Hash: `A1BE60039F125080560EDF1EEBEE5B6D9E2D6039F5F5AC478E6273E05EDADB4B`
+- SHA256 Hash: `6FCDEC03A8E91D77D4682E9839622D8C42F5B47BD0DCD19DDC7F5F4764EDB7DA`
```
Comment (he3als) [inline src/AtlasModules/README.md:5]: This should be the hash of the executable (`devmanview.exe`), not the `.zip` that it is downloaded in.

Hash of the executable: `fc81134cfb57cd64a76beeac74cfab29768d3b542ec6b25429f5337780790a1e`

The updated executable in this pull request does **not** match this though, and it has a file size difference of 50kb.

explorer_W0CNaL6O4O

**Review by he3als — CHANGES_REQUESTED** [review]
All of the hashes match, except from the updated executable for DevManView.

File: src/AtlasModules/README.md (hunk 4 above)
Comment (JayXTQ) [reply src/AtlasModules/README.md:5]: Thought I had the correct one, thanks for the recorrection, will fix now!

**Comment by JayXTQ** [comment]
Fixed wrong hashes par @he3als

File: src/AtlasModules/README.md (hunk 4 above)
Comment (he3als) [reply src/AtlasModules/README.md]: Fixed

**Review by he3als — APPROVED** [review]

**Pull request merged**